openfl/pipelines/__init__.py

```diff
@@ -3,6 +3,7 @@
 """openfl.pipelines module."""
 
 from .kc_pipeline import KCPipeline
+from .eden_pipeline import EdenPipeline
```

Comment:
This import is failing because `torch` is not installed by default in OpenFL. Could you make this a conditional import as follows (see [here](https://github.com/intel/openfl/blob/develop/openfl/federated/task/__init__.py) for how we handle this in our task runners):

```
import pkgutil
...
if pkgutil.find_loader('torch'):
    from .eden_pipeline import EdenPipeline  # NOQA